File: problem/Triangular_Sum_Array/triangularSum.py
class Solution:
    def triangularSum(self, nums):
        n = len(nums)
        for size in range(n, 1, -1):
            for i in range(size - 1):
                nums[i] = (nums[i] + nums[i + 1]) % 10
        return nums[0]

File: problem/Triangular_Sum_Array/triangularSum.c
int triangularSum(int *nums, int numsSize)
{
    for (int n = numsSize; n > 1; --n)
    {
        for (int i = 0; i < n - 1; ++i)
        {
            nums[i] = (nums[i] + nums[i + 1]) % 10;
        }
    }
    return nums[0];
}